lib/cli/run-binary.js

```diff
@@ -8,11 +8,17 @@ module.exports = function(args, options, types) {
 
   return utils.projectRoot()
     .then(function(root) {
-      var doneScript = process.platform === 'win32' ? 'donejs.cmd' : 'donejs';
+      var doneScript = process.platform === 'win32' ? 'donejs-cli.cmd' : 'donejs-cli';
       var donejsBinary = path.join(root, 'node_modules', '.bin', doneScript);
 
+      // backwards compatiblilty for bin/donejs before it became bin/donejs-cli
       if (!fs.existsSync(donejsBinary)) {
-        var msg = 'Could not find local DoneJS binary (' + donejsBinary + ')';
+        doneScript = process.platform === 'win32' ? 'donejs.cmd' : 'donejs';
```

Comment:
Yes, this test proves the backwards compatibility: https://github.com/donejs/donejs/blob/f3187f7874ad981f19a55df78493c15cadda4f8a/test/run-binary-test.js#L70-L98